src/config/vcap-services.js

```diff
@@ -9,17 +9,25 @@ function getVCAPServices (service) {
   return services
 }
 
-function getVariableService (userProvidedServices) {
-  const variableServices = userProvidedServices.filter(service => service.instance_name === 'variable-service')
-  if (variableServices.length !== 1) {
-    throw new Error(`Expected exactly one variable-service, instead found ${variableServices.length}`)
+function getService (userProvidedServices, serviceName) {
+  const services = userProvidedServices.filter(service => service.instance_name === serviceName)
+  if (services.length !== 1) {
+    throw new Error(`Expected exactly one variable-service, instead found ${services.length}`)
```

Comment:
You should change the reference to the `variable-service` in the message too.

Reply:
Updated error message.